praw/config.py

```diff
@@ -31,9 +35,15 @@ def _config_boolean(item):
         return item.lower() in {"1", "yes", "true", "on"}
 
     @classmethod
-    def _load_config(cls):
+    def _load_config(cls, config_interpolation: str = None):
```

Comment:
Shouldn't this be `Optional[str]` due to the default being `None`?

Reply:
Actually, the type annotation probably doeesn't matter since it's a private method.